app/index.js

```diff
@@ -742,3 +709,56 @@ if (DEBUG) {
         return this.modulesByType(type)[0];
     };
 }
+
+/**
+ *
+ * @param {string} parentId
+ * @return {string|*}
+ * @private
+ */
+Application.prototype._nextModuleId = function(parentId) {
+    var key = parentId || 'root';
+    if (!(key in this._ids)) {
+        this._ids[key] = 1;
+    }
+    var nextId = this._ids[key]++;
+    return _.compact([parentId, nextId]).join('-');
+};
+
+/**
+ * Setup plugins.
+ *
+ * @param {Array} plugins list of plugins
+ * @private
+ */
+Application.prototype._setupPlugins = function(plugins) {
+    _.each(plugins, function(name) {
+        var pluginModule = lookupRequire('plugins', name);
+        this[name] = pluginModule(this);
+    }, this);
+};
+
+/**
+ * Requires some specific entity by its location and name.
+ * Seeks the entity in end-user application then seeks it in the framework.
+ *
+ * @param {string} location The entity location
+ * @param {string} name Module name to load
+ * @return {*}
+ */
+function lookupRequire(location, name) {
+    var modulePath = location + '/' + name;
```

Comment:
Unfortunately we have tons of code that doesn't support windows :( Won't fix right now.